test/useObservable.test.tsx

```diff
@@ -76,7 +76,7 @@ describe("is used to keep observable within component body", () => {
                 x: 1,
                 y: 2,
                 get z() {
-                    return obs.x + obs.y
+                    return this.x + this.y

```

Comment:
Please revert this, it's intentional like this. You should be able to access values without referencing `this`.